Dockerfile

```diff
@@ -11,7 +11,14 @@ RUN go mod download
 
 COPY . .
 
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o kubestr .
```

Comment:
Shouldn't this (and doesn't this already) happen outside the dockerfile?

Reply:
`jobs:
  # Run tests.
  # See also https://docs.docker.com/docker-hub/builds/automated-testing/
  test:
    runs-on: ubuntu-latest

    steps:
      - name: Check out code into the Go module directory
        uses: actions/checkout@v2

      - name: Run tests
        run: docker build . --file Dockerfile`

Reply:
Forgive the formatting but no it doesn't. The steps to deploy are to first download the repo and then run docker build.